### ISSUE TITLE
Questions about learning rate and batch size

### ISSUE THREAD

**Issue by Bin-ze** [open-mmlab/mmdetection#5504]
In the general configuration, Lr=0.02, the batch size on each GPU is 16, your training plan uses 8 GPUs, I use four GPUs for training, does it mean that my batch size is the second of when I use 8 blocks One part, still said that it will not be affected?
   I found that when I use 4 GPUs, the network training and convergence speed is very fast, but when I use two GPUs to retrain from the breakpoint, the loss increases than before, and the training speed is about 10 times slower. Logically speaking, it should be about twice. Another point is that I use two cards for training, do I need to set Lr to half of the 4 cards?

**Comment by jshilong** [comment]
>  Another point is that I use two cards for training, do I need to set Lr to half of the 4 cards?
sure

**Comment by Bin-ze** [comment]
> > Another point is that I use two cards for training, do I need to set Lr to half of the 4 cards?
> 
> yes

It means that if the lr of 8 cards is 0.02, then I need to set it to 0.005 when I use 2 fast cards?

**Comment by Bin-ze** [comment]
It is reasonable that the setting of distributed training should not be so, because I think the data on each card is only shared when calculating loss, and is isolated at other times, so why is the total batch size numGPU*oneGPU batch?

**Comment by jshilong** [comment]
> It is reasonable that the setting of distributed training should not be so, because I think the data on each card is only shared when calculating loss, and is isolated at other times, so why is the total batch size numGPU*oneGPU batch?

Could you explain your question more clearly?

**Comment by jshilong** [comment]
Feel free to reopen the issue if there is any question
